plugin/src/main/resources/io/jenkins/plugins/analysis/core/model/ReportScanningTool/help-contextLines.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
Can you move that to help-linesLookAhead.html, then we do not need to specify the name in the entry.